app/models/event.rb

```diff
@@ -29,6 +31,10 @@ def to_s
     self.name
   end
 
+  def next_upcoming(n)
+    Event.order('date_and_time').where('date_and_time > ?', Date.today).take(3) 
```

Comment:
you're completely right, I was attempting to write a class method here (failed) and failed to remove this after writing the scope